src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/collection/CollectionFilter.java

```diff
@@ -123,7 +123,7 @@ protected static JsonNode toJsonNode(JsonNodeFactory nodeFactory, List<Object> l
 
   protected static JsonNode toJsonNode(JsonNodeFactory nodeFactory, Map<String, Object> mapValues) {
     final ObjectNode objectNode = nodeFactory.objectNode();
-    mapValues.forEach((k, v) -> objectNode.put(k, toJsonNode(nodeFactory, v)));
+    mapValues.forEach((k, v) -> objectNode.set(k, toJsonNode(nodeFactory, v)));
```

Comment:
Checking this change from objectNode.put() to objectNode.set() was deliberate

Reply:
Yes -- "put" gave warning, being Deprecated; "set" is the intended replacement (that or "replace").